huaweicloud/services/acceptance/dli/resource_huaweicloud_dli_flinkjar_job_test.go

```diff
@@ -0,0 +1,248 @@
+package dli
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/chnsz/golangsdk/openstack/dli/v1/flinkjob"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils/fmtp"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+func getDliFlinkJarJobResourceFunc(config *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := config.DliV1Client(acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmtp.Errorf("error creating Dli v1 client, err=%s", err)
+	}
+	jobId, _ := strconv.Atoi(state.Primary.ID)
+	return flinkjob.Get(client, jobId)
+}
+
+func TestAccResourceDliFlinkJarJob_basic(t *testing.T) {
+	var obj flinkjob.CreateJarJobOpts
+	resourceName := "huaweicloud_dli_flinkjar_job.test"
+	name := acceptance.RandomAccResourceName()
+	bucketName := acceptance.RandomAccResourceNameWithDash()
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&obj,
+		getDliFlinkJarJobResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckDliJarPath(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFlinkJarJobResource_basic(name, bucketName, acceptance.HW_REGION_NAME,
+					acceptance.HW_ACCESS_KEY, acceptance.HW_SECRET_KEY, acceptance.HW_DLI_FLINK_JAR_OBS_PATH),
```

Comment:
There is no need to pass environment variables in the parameter list of the function.

Reply:
keep it as it is.  function focus logic , ignore how to get arguments

Reply:
I don't agree it.